CHANGELOG.md

```diff
@@ -11,6 +11,122 @@ git log --first-parent
+
+### Added features
+
++ Add a pipelined CG solver [#1824](https://github.com/ginkgo-project/ginkgo/pull/1824), [#1838](https://github.com/ginkgo-project/ginkgo/pull/1838), [#1859](https://github.com/ginkgo-project/ginkgo/pull/1859)
++ Add Coo Transpose/Conj-Transpose [#1816](https://github.com/ginkgo-project/ginkgo/pull/1816)
++ Add chebyshev iteration solver [#1289](https://github.com/ginkgo-project/ginkgo/pull/1289)
++ Add a two-level Schwarz preconditioner [#1431](https://github.com/ginkgo-project/ginkgo/pull/1431)
++ Add simplified configuration for stopping criteria [#1613](https://github.com/ginkgo-project/ginkgo/pull/1613)
++ Add an example to show the distributed multigrid usage [#1769](https://github.com/ginkgo-project/ginkgo/pull/1769)
++ Add enable half in MPI [#1759](https://github.com/ginkgo-project/ginkgo/pull/1759)
++ Add yaml-cpp reader for config file in extension [#1677](https://github.com/ginkgo-project/ginkgo/pull/1677)
++ Add sequential and distributed L1-Jacobi [#1310](https://github.com/ginkgo-project/ginkgo/pull/1310), [#1806](https://github.com/ginkgo-project/ginkgo/pull/1806)
++ Add reusable permutation and transpose operations [#1338](https://github.com/ginkgo-project/ginkgo/pull/1338)
++ Add collective communication interface and dense/neighborhood implementation of the interface [#1780](https://github.com/ginkgo-project/ginkgo/pull/1780)
++ Add local-to-global index mapping [#1707](https://github.com/ginkgo-project/ginkgo/pull/1707)
++ Add Minres solver [#975](https://github.com/ginkgo-project/ginkgo/pull/975)
++ Add `array::copy_to_host` utility function [#1835](https://github.com/ginkgo-project/ginkgo/pull/1835)
++ Add bfloat16 support and corresponding MPI function [#1825](https://github.com/ginkgo-project/ginkgo/pull/1825), [#1827](https://github.com/ginkgo-project/ginkgo/pull/1827)
++ Add mixed precision support for distributed matrix when the underlying matrix also support mixed precision [#1819](https://github.com/ginkgo-project/ginkgo/pull/1819).
++ Add distributed RowGatherer which is used by the distributed matrix to handle the communication [#1589](https://github.com/ginkgo-project/ginkgo/pull/1589)
++ Add complex type support for Dense transpose and Fbcsr on AMD GPUs [#1839](https://github.com/ginkgo-project/ginkgo/pull/1839)
++ Add OMP implementation for Merge-Path CSR [#1810](https://github.com/ginkgo-project/ginkgo/pull/1810)
+
+### Improvements
+
++ Improve performance of factorization validation in benchmarks [#1766](https://github.com/ginkgo-project/ginkgo/pull/1766)
++ Allow the ValueType additionally in precondition Ic [#1811](https://github.com/ginkgo-project/ginkgo/pull/1811) and Ilu [#1828](https://github.com/ginkgo-project/ginkgo/pull/1828) and Ilu [#1828](https://github.com/ginkgo-project/ginkgo/pull/1828). Note. It introduces the behavior changes for config usage. Please take a look at the behavior changes section.
```

Comment:
```suggestion
+ Allow specifying a ValueType instead of a full SolverType in preconditioners Ic [#1811](https://github.com/ginkgo-project/ginkgo/pull/1811) and Ilu [#1828](https://github.com/ginkgo-project/ginkgo/pull/1828) and Ilu [#1828](https://github.com/ginkgo-project/ginkgo/pull/1828). Note. It introduces the behavior changes for config usage. Please take a look at the behavior changes section.
```